Channels/Production/release_notes.md

```diff
@@ -1,27 +1,28 @@
-## April 12, 2021 Production Release ([v1.1.1562.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.1562.01))
+## July 21 2021 Production Release ([v1.1.1662.02](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.1662.02))
 
-Welcome to the April 12, 2021 Production release of Accessibility Insights for Windows.
+Welcome to the July 21 2021 Production release of Accessibility Insights for Windows.
 
-Installation Link: https://github.com/microsoft/accessibility-insights-windows/releases/download/v1.1.1562.01/AccessibilityInsights.msi
+Installation Link: https://github.com/microsoft/accessibility-insights-windows/releases/download/v1.1.1662.02/AccessibilityInsights.msi
 
 Documentation Link: https://accessibilityinsights.io/docs/en/windows/overview
 
 ### Highlights
 
+- [New Features](#new-features)
 - [Rule Updates](#rule-updates)
 - [Bug Fixes](#bug-fixes)
 
+#### New Features
+
+- Added support for inspecting and saving [Custom UIAutomation properties](https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-propertiesoverview). Details on how to configure this feature can be found [in the online documentation](https://accessibilityinsights.io/docs/en/windows/reference/faq/#does-accessibility-insights-for-windows-support-custom-ui-automation-properties).
+
 #### Rule Updates
 
-- Add the `FrameworkDoesNotSupportUIAutomation` rule to report failures on frameworks that are known to not support UI AUtomation [#554](https://github.com/microsoft/axe-windows/issues/554)
-- Remove the `ControlShouldNotSupportTablePattern` rule as it wasn't generally useful [#566](https://github.com/microsoft/axe-windows/issues/566)
-- Remove false positives in the `BoundingRectangleCompletelyObscuresContainer` rule [#540](https://github.com/microsoft/axe-windows/issues/540)
-- Remove false positives in the `NameIsNotNull` rule [#556](https://github.com/microsoft/axe-windows/issues/556)
+- `TextBlock` contents inside `Button` elements are no longer flagged in the NameExcludesPrivateUnicodeCharacters rule for WPF. ([#572](https://github.com/microsoft/axe-windows/issues/572))
+- The implementation of `TogglePattern` on radio button elements is no longer flagged as an error due to the removal of the `ControlShouldNotSupportTogglePattern` rule. ([#590](https://github.com/microsoft/axe-windows/issues/590))
```

Comment:
did we remove the rule completely?

Reply:
Yes, because it only ran on radio buttons anyway.